projects/agave.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     ]),
     borrowed: sdk.util.sumChainTvls([
       aaveChainTvl("xdai", addressesProviderRegistryXDAI, addr => `xdai:${addr}`, [protocolDataHelper], true),
-      aaveChainTvl("xdai", addressesProviderRegistryXDAI_old, addr => `xdai:${addr}`, [protocolDataHelper_old], true),
+      // aaveChainTvl("xdai", addressesProviderRegistryXDAI_old, addr => `xdai:${addr}`, [protocolDataHelper_old], true), // borrowed amount is lost due to attack
```

Comment:
@g1nt0ki i would use a function that count is before the hack but not after, that way it can be refilled

Reply:
fixing it